src/context/NotificationContext.tsx

```diff
@@ -2,77 +2,76 @@ import React, { useState, useCallback } from 'react';
 import Notification, { INotificationProps } from '../Alerts/atom/Notification';
 
 export type NotificationContextType = {
-  sendNotification: (newNotification : INotificationProps) => void;
+  sendNotification: (newNotification: INotificationProps) => void;
 };
 
 const defaultContext: NotificationContextType = {
-  sendNotification : () => console.log("This is the context initialization should not appear"),
-}
+  sendNotification: () => console.log("This is the context initialization should not appear"),
+};
 
-const NotificationContext = React.createContext<NotificationContextType>(defaultContext);
+export const NotificationContext = React.createContext<NotificationContextType>(defaultContext);
```

Comment:
I tend to feel it's better practice to group the exports rather than having one in the middle of code. So perhaps in this instance it's good to declare the `const` here andexport this and any others at the end of the file.

Reply:
I don't have a preference for this so I'll go ahead an update